packages/smooth_app/lib/pages/product/edit_ingredients_page.dart

```diff
@@ -292,7 +311,10 @@ class _OcrWidget extends StatelessWidget {
                 ),
                 child: SmoothActionButtonsBar(
                   positiveAction: SmoothActionButton(
-                    text: helper.getActionRefreshPhoto(appLocalizations),
+                    text: (hasImageProvider ||
+                            helper.getImageUrl(product) != null)
+                        ? helper.getActionRefreshPhoto(appLocalizations)
+                        : 'Upload Photo',
```

Comment:
Should be localized.

Reply:
Localization can be done by adding a new string to `lib/l10n/app_en.arb`  then you need to run flutter pub get (or sometimes run the app) and you should be able to access It using the appLocalization object we already have in this screen

Reply:
Thanks @M123-dev